src/Microsoft.DotNet.Interactive.SqlServer.Tests/MsSqlServiceClientTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+using FluentAssertions;
+using System.IO;
+
+namespace Microsoft.DotNet.Interactive.SqlServer.Tests
+{
+    public class MsSqlServiceClientTests
+    {
+        [Fact]
+        public void Should_parse_doc_change_correctly_with_windows_line_endings()
+        {
+            RunDocumentChangeTest("\r\n");
+        }
+
+        [Fact]
+        public void Should_parse_doc_change_correctly_with_unix_line_endings()
+        {
+            RunDocumentChangeTest("\n");
+        }
+
+        private void RunDocumentChangeTest(string lineEnding)
+        {
+            string oldText = string.Join(lineEnding, "abc", "def", "", "abc", "abcdef");
+            int oldTextLineCount = 5;
+            int oldTextLastCharacterNum = 6;
+            string newText = string.Join(lineEnding, "abc", "def");
+            var testUri = new Uri("untitled://test");
+
+            var docChange = MsSqlServiceClient.GetDocumentChangeForText(testUri, newText, oldText);
+
+            docChange.ContentChanges.Length
+                .Should()
+                .Equals(1);
```

Comment:
This is a common `FluentAssertions` gotcha. You should make this change throughout these. The `Equals` call here is `Object.Equals` which returns false but doesn't throw, so you'll get false positives.

```suggestion
                .Be(1);
```